src/main/java/seedu/duke/commands/member/TeamMembersListCommand.java

```diff
@@ -3,26 +3,24 @@
 import seedu.duke.commands.Command;
 import seedu.duke.member.TeamMember;
 import seedu.duke.project.Project;
-import seedu.duke.project.ProjectList;
-import seedu.duke.member.TeamMemberList;
 
 import java.util.ArrayList;
 
-public class ListTeamMembersCommand extends Command {
+public class TeamMembersListCommand extends Command {
 
     private String name;
 
-    public ListTeamMembersCommand() {
+    public TeamMembersListCommand() {
     }
 
-    public String executeCommand(ArrayList<Project> projects) {
-        if (Project.getMembers().size() == 0) {
+    public String executeCommand(ArrayList<Project> projects, ArrayList<TeamMember> teamMembers) {
+        if (teamMembers.size() == 0) {
             return "No team members have been added.";
         }
 
         String listOfMembers = "";
         int i = 0;
-        for (TeamMember member : Project.getMembers()) {
+        for (TeamMember member : teamMembers) {
             String memberLine = (i + 1) + ". " + member.getName() + "\n";
             listOfMembers += memberLine;
             i++;
```

Comment:
Are we printing lists here or in Ui? I actually don't mind either but should be standardised. Not a high priority as of now though.

Reply:
I have no preference, I think both make when it comes to organisation